store/tikv/coprocessor.go

```diff
@@ -1003,6 +1011,35 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *RPCCon
 	return nil, nil
 }
 
+func (worker *copIteratorWorker) handleTiDBSendReqErr(err error, task *copTask, ch chan<- *copResponse) error {
+	errCode := 0
+	errMsg := err.Error()
+	if terror.ErrorEqual(err, ErrTiKVServerTimeout) {
```

Comment:
Could the error be something else other than `ErrTiKVServerTimeout`?
errCode = 0 seems not a good default, how about using unknown error code?

Reply:
great, done.